app/src/main/java/org/mifos/mobile/ui/activities/base/BaseActivity.java

```diff
@@ -73,10 +72,7 @@ protected void onCreate(Bundle savedInstanceState) {
      */
     public ActivityComponent getActivityComponent() {
         if (activityComponent == null) {
-            activityComponent = DaggerActivityComponent.builder()
```

Comment:
Removing this will affect the working of the application. Please undo the changes